src/core/declarativewebcontainer.cpp

```diff
@@ -743,11 +746,14 @@ void DeclarativeWebContainer::initialize()
         return;
     }
 
+    // From this point onwards, we're ready to initialize.
```

Comment:
We're not yet initialized as init is finalized within if-else if block right after this. As we might end up calling ```loadTab``` and it is guarded by ```m_initialized``` we need to set it before.